docs/device/update.md

```diff
@@ -0,0 +1,18 @@
+# Updating naisdevice
+
+These instructions assume that you already have some version of naisdevice installed, and that you are already familiar with the general method of use.
+If this is not the case, head over to the [installation instructions](./install.md).
```

Comment:
Add a caveat that if something goes wrong with the update process, they can just re-run the installation from the "download naisdevice" step (not the "register device with Kolide" step)?